Maddori.Apple/Maddori.Apple/Screen/Home/SelectReflectionMember/ViewController/SelectReflectionMemberViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  SelectFeedbackMemberViewController.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/25.
+//
+
+import UIKit
+
+import Alamofire
+import RxSwift
+
+final class SelectReflectionMemberViewController: BaseViewController {
+    
+    // MARK: - property
+    
+    private var reflectionId: Int = 0
+    private let selectReflectionMemberView = SelectReflectionMemberView()
+    
+    private let viewModel: any ViewModelType
+    private let disposeBag: DisposeBag = DisposeBag()
+    
+    // MARK: - life cycle
+    
+    init(viewModel: any ViewModelType) {
+        self.viewModel = viewModel
+        super.init()
+    }
+    
+    required init?(coder: NSCoder) { nil }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        bindView()
+        bindViewModel()
+    }
+    
+    override func loadView() {
+        view = selectReflectionMemberView
+    }
+    
+    // MARK: - override
+    
+    override func setupNavigationBar() {
+        super.setupNavigationBar()
+        configureNavigationController()
+    }
+    
+    // MARK: - private func
+    
+    private func configureNavigationController() {
+        guard let navigationController else { return }
+        selectReflectionMemberView.setupNavigationController(navigationController)
+        selectReflectionMemberView.setupNavigationItem(navigationItem)
+    }
+    
+    private func bindViewModel() {
+        let output = transformInput()
+        bind(output: output)
+    }
+    
+    private func transformInput() -> SelectReflectionMemberViewModel.Output? {
+        guard let viewModel = viewModel as? SelectReflectionMemberViewModel else { return nil }
+        let input = SelectReflectionMemberViewModel.Input(viewDidLoad: Observable.just(()).asObservable(), viewWillAppear:  rx.sentMessage(#selector(UIViewController.viewWillAppear(_:))).map{ _ in }.asObservable(), didTappedFeedbackDoneButton: selectReflectionMemberView.feedbackDoneButtonTapPublisher)
```

Comment:
p1;
요부분 어제 머지한 제 PR ( Rx+Extension 관련된거 ) 땡기면 아래처럼 바꿀 수 있어유 ( 아마 )
```suggestion
        let input = SelectReflectionMemberViewModel.Input(
            viewDidLoad: self.rx.viewDidLoad.asObservable(), 
            viewWillAppear:  self.rx.viewWillAppear.asObservable(), 
            didTappedFeedbackDoneButton: selectReflectionMemberView.feedbackDoneButtonTapPublisher
        )
```

Reply:
추가로 
요 부분은 개행 처리를 해서 가독성을 높이면 어떨까 제안드립니다.
<img width="1439" alt="스크린샷 2023-12-29 오후 8 49 39" src="https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/assets/59243274/3049821c-a2f2-4441-8a28-804a8ecb23a2">

Reply:
[9daf6d9](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/407/commits/9daf6d9485a066f893f6fc5a96540944ca74544b)
풀을 안 받았군요 ~ 변경 완료 ⭐️